go/cmd/dolt/commands/diff.go

```diff
@@ -438,6 +431,43 @@ func diffUserTables(ctx context.Context, fromRoot, toRoot *doltdb.RootValue, dAr
 	return nil
 }
 
+func newSqlEngine(ctx context.Context, dEnv *env.DoltEnv) (*engine.SqlEngine, error) {
```

Comment:
lots of other cli commands will probably want to use the engine. Maybe this func should live in the commands/engine package?

Reply:
Good call, moved